src/main/java/gov/usgs/wma/waterdata/TimeSeriesDao.java

```diff
@@ -38,4 +46,9 @@ public Object pruneTimeSeries(String date) {
 		}
 		return result;
 	}
+
+	public String processDate(String date) {
```

Comment:
I thought we were using jackson to have the request be a LocalDate and passing it as a LocalDate thru the program?

Reply:
I struggled to get that working early-on - but it is now implemented as we discussed.